Libraries/Network/RCTNetworking.m

```diff
@@ -515,6 +573,32 @@ - (RCTNetworkTask *)networkTaskWithRequest:(NSURLRequest *)request
   [_tasksByRequestID removeObjectForKey:requestID];
 }
 
+#pragma mark - Decoding Helper
+
+static NSMutableDictionary *undecodablePartialDataPool;
```

Comment:
This would need to be associated with the RCTNetworkTask, not by adding a global.